practices/advertising.md

```diff
@@ -0,0 +1,54 @@
+# Advertising
+
+These guidelines are based on ethical, [accessibility](accessibility.md) or performance concerns.
```

Comment:
Replacing "concerns" with "recommendations" will be more in line with your aim to keep the wording positive here